python/ml_croissant/scripts/validate.py

```diff
@@ -6,8 +6,7 @@
 from absl import flags
 from absl import logging
 
-from ml_croissant import Dataset
-from ml_croissant import ValidationError
+import ml_croissant as mlc
```

Comment:
Unrelated to this PR, but maybe it might be useful to have a more transparent script description than just "validate script"?